code/Silvergraph.php

```diff
@@ -51,7 +60,7 @@ public function dot(){
         $renderClasses = array();
 
         //Get all DataObject subclasses
-        $dataClasses = ClassInfo::subclassesFor('DataObject');
+        $dataClasses = ClassInfo::subclassesFor('SilverStripe\\ORM\\DataObject');
```

Comment:
You could do this with `DataObject::class`

Reply:
:+1: